draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -402,7 +399,12 @@ JSON:
   EDE option that conveys the information in the "c", "s", and "j"
   fields encoded in the JSON object.
 
+* If the local application that triggered the DNS request has more specific
+  context (e.g., contact information) that the DNS server, the content of
```

Comment:
I think you mean the following:

"If the local application that triggered the DNS request has more specific context (e.g., contact information) about the DNS server...".

Reply:
I meant "than the DNS server"